lib/Core/TxTree.h

```diff
@@ -708,6 +708,7 @@ class TxTree {
   static Statistic executeMemoryOperationTime;
   static double entryNumber;
   static double programPointNumber;
+  static uint64_t blockCount;
```

Comment:
@rasoolmaghareh Perhaps this should be declared after `subsumptionCheckCount`, following the initialization order in `TxTree.cpp`.